shared/chat/conversation/messages/react-button/emoji-picker/index.js

```diff
@@ -138,7 +139,7 @@ const EmojiRow = ({
 
 const EmojiRender = ({emoji, onChoose}: {emoji: EmojiData, onChoose: EmojiData => void}) => (
   <ClickableBox onClick={() => onChoose(emoji)} style={styles.emoji} key={emoji.short_name}>
-    <Emoji size={singleEmojiWidth} emojiName={`:${emoji.short_name}:`} />
+    <Emoji size={isAndroid ? singleEmojiWidth - 5 : singleEmojiWidth} emojiName={`:${emoji.short_name}:`} />
```

Comment:
The default google emoji font renders about 5 units larger than the font size, so this is a workaround to make those fit. Other emoji fonts are at risk of being clipped by a bounding box here if their sizing is larger than google's.